tools/packages/wesl-link/src/cli.ts

```diff
@@ -57,28 +57,25 @@ function parseArgs(args: string[]) {
 }
 
 async function linkNormally(paths: string[]): Promise<void> {
+  const weslRoot = getBaseDir();
   const pathAndTexts = paths.map(f => {
     const text = fs.readFileSync(f, { encoding: "utf8" });
-    const relativePath = path.relative(process.cwd(), f);
-    const basedPath = "./" + normalize(relativePath);
-    return [basedPath, text];
+    const relativePath = path.relative(weslRoot, f);
+    return [toUnixPath(relativePath), text];
```

Comment:
Always carefully convert to unix style, relative paths when it ends up being a linker argument.